src/sass/base/_fonts.scss

```diff
@@ -0,0 +1,34 @@
+@font-face {
+  font-family: 'TitanOne';
+  src: url(../fonts/TitanOne-Regular.ttf) format('ttf'),
```

Comment:
please change all filenames to lowercase

Reply:
@izachok fixed😀